efi/preinstall/checks_context.go

```diff
@@ -548,7 +549,58 @@ func (c *RunChecksContext) runAction(action Action, args ...any) error {
 
 		return NewWithKindAndActionsError(kind, nil, errorKindToActions[kind], err)
```

Comment:
@ernestl do we have a problem that in you design args for actions are a map (like for errors) but the code actually takes []any ?